packages/client/src/views/MyGrantsView.vue

```diff
@@ -7,20 +7,27 @@
       style="margin-top: 1.5rem"
       lazy
     >
-      <b-tab title="Interested">
+      <b-tab v-if="shareTerminologyEnabled" title="Shared With Your Team">
```

Comment:
Have you set up linting and auto-formatting in your editor yet? You should hopefully be able to get running with auto-fixing for lint errors like what's showing up on this and the line below. 

Bit of documentation is here, but I'm happy to help as well if you need! https://github.com/usdigitalresponse/usdr-gost/blob/main/docs/development.md#linting

Reply:
That aside, given that the functionality doesn't really change, but only the tab title, it's a bit more DRY here to just make the titles dynamic: 

```js
<b-tab :title="shareTerminologyEnabled ? 'Shared With Your Team' : 'Assigned'">
  <GrantsTable
    :search-title="shareTerminologyEnabled ? 'Shared With Your Team' : 'Assigned'"
    :show-assigned-to-agency="selectedAgencyId"
    :show-search-controls="false"
  />
</b-tab>
```

You could also turn the `shareTerminologyEnabled ? 'Shared With Your Team' : 'Assigned'` into a computed property if you want, but one repetition I'm on the fence about whether it's worth it.

Reply:
Thank you for the detailed response and code suggestions!